### PR TITLE
New supported-platforms@es.md file

### PR DESCRIPTION
**Pull request by SidVal**
New file for ../es/support/faqs/supported-platforms

***
I allow edits from maintainers; feel free to modify if you need it.

### CODE REVIEW

File: content/support/faqs/supported-platforms@es.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
+---
+$title@: Supported Platforms, Vendors and Partners
```
Comment (bpaduch) [inline content/support/faqs/supported-platforms@es.md]: Can this title be translated? Also remove the `@`.

File: content/support/faqs/supported-platforms@es.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+$order: 3
+$parent: /content/support/faqs.md
+class: who
+
+description@: The Accelerated Mobile Pages (AMP) Project is an open source initiative that makes it easy for publishers to create mobile-friendly content once and have it load instantly everywhere. – Accelerated Mobile Pages Project
```
Comment (bpaduch) [inline content/support/faqs/supported-platforms@es.md]: Please translate this description (and remove the `@`).

File: content/support/faqs/supported-platforms@es.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+
+cta:
+  title@: Próximo FAQ
```
Comment (bpaduch) [inline content/support/faqs/supported-platforms@es.md]: Remove `@`

File: content/support/faqs/supported-platforms@es.md [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+  link_text@: AMP Overview
```
Comment (bpaduch) [inline content/support/faqs/supported-platforms@es.md]: Please translate this description (and remove the `@`).

Comment (SidVal) [inline content/support/faqs/supported-platforms@es.md]: I think `AMP Overview` does not must be translated. There is not a word in spanish for "overview".

**Comment by mrjoro** [comment]
@SidVal I'm working on a project to recognize people who have contributed to AMP and would like to include you.  It requires an email address and I don't have access to yours; if you feel comfortable doing so can you send me your email address (either email it to my username @ampproject.org or on DM me on the [AMP Slack](https://amphtml.slack.com/messages/C0ADR0A0K/) @mrjoro?

**Comment by SidVal** [comment]
Hi @mrjoro, sorry the lag for response to you. :mailbox_closed:  

I wrote an email to you.

Regards

**Comment by mrjoro** [comment]
Thanks @SidVal!

**Pull request merged**